src/__tests__/modals/__snapshots__/AccelerateTxModal.test.tsx.snap

```diff
@@ -53,7 +53,7 @@ exports[`AccelerateTxModalComponent should render with loading props 1`] = `
       showSpinner={false}
     />
   </View>
-  <ModalCloseArrow
```

Comment:
Do not make this a fixup commit. Renames should be a commit of their own.

Reply:
Now a seperate commit